src/components/Inventory/view/BatchesCellRenderer.tsx

```diff
@@ -20,8 +20,9 @@ export default function BatchesCellRenderer(props: RendererProps<TableRowType>):
   const { column, row, value, index, onRowClick } = props;
 
   const withdraw = () => {
-    // TODO
-    return;
+    if (onRowClick) {
```

Comment:
here and for edit/click... we may need to prevent event propagation (row gets selected otherwise)?

Reply:
I tried doing it here but it's not working. Maybe we need to do it on the table component

Reply:
we can follow up on this later maybe, someone will surely file a bug :)

Reply:
Sounds good!

Reply:
I took care of it for now https://github.com/terraware/terraware-web/pull/665